src/containers/index.tsx

```diff
@@ -28,6 +28,50 @@ import EmployeeEdits from './EmployeeEdits';
 import AddCheckInOut from './AddCheckInOut';
 
 const Router: FC = () => {
+  const location = useLocation();
+  useEffect(() => {
+    // Update document title on initial render
+    document.title = 'EMS: '.concat(getPageTitle(location.pathname));
+  }, [location.pathname]);
+
+  const getPageTitle = (pathname: string) => {
```

Comment:
Better approach here would be to create a custom route component
```
import { Route } from 'react-router-dom';
import PropTypes from 'prop-types';

export const Page = ({ title, ...rest }) => {
  useEffect(() => {
    document.title = 'EMS: '.concat(getPageTitle(location.pathname));
  }, [title]);
  return <Route {...rest} />;
};
```

and use it as 

```
<Page
    path="/employee"
    element={<Employees />}
    title="Employee Details"
  />
```

insteead of 
`<Route path="/employee" element={<Employees />} />`

This way you won't have to maintain the switch case by yourself.